CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## v0.4.0 (not released yet)
 
+* New flags `-collect.database` and `-collect.collection` can be used to enable collection of database and collection
```

Comment:
Was this somewhere requested? Jira issue?

Reply:
That's #84.